MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +287,30 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        let threshold = 1000.0 //meters
+        let zoomOutAltitude = 2000.0
+        let zoomInAltitude = 1000.0
+        
+        let isZoomedOut = abs(mapView.camera.altitude - 2000) <= 25
```

Comment:
What do the 2000 and 25 values mean here?

Reply:
Since altitude is a float, I noticed that the altitude is not exactly `2000.0000...`, (which is not unusual in the floating point world) so it's basically just a tolerance to ask "are we within 25 meters of our 'zoomed out' altitude?" That tolerance should probably be shaved down.